index.html

```diff
@@ -1,12 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <meta
+    http-equiv="X-UA-Compatible"
+    content="IE=edge"
+  >
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1.0"
+  >
+  <title>calculator</title>
+
+  <link rel="stylesheet" href="./styles.css">
 </head>
+
 <body>
+
+  <div class="calculator">
```

Comment:
That looks quite nice and readable

if you want, you can apply BEM classes http://getbem.com/

Reply:
BEM added